app/scripts/modules/kubernetes/pipeline/stages/runJob/runJobStage.html

```diff
@@ -19,17 +19,40 @@
   </stage-config-field>
 
   <stage-config-field label="Image">
-    <docker-image-and-tag-selector
-      specify-tag-by-regex="false"
-      account="ctrl.stage.container.imageDescription.account"
-      organization="ctrl.stage.container.imageDescription.organization"
-      registry="ctrl.stage.container.imageDescription.registry"
-      repository="ctrl.stage.container.imageDescription.repository"
-      tag="ctrl.stage.container.imageDescription.tag"
-      show-registry="true"
-      on-change="ctrl.onChange"></docker-image-and-tag-selector>
+    <div class="checkbox">
+      <label>
+        <input type="checkbox"
+               ng-model="ctrl.stage.container.imageDescription.fromTrigger"/>
+        Resolve from Trigger
```

Comment:
This might look a little better if the checkbox came after `Resolve from Trigger`, and if there was some vertical space between this field and the fields below it.